style.css

```diff
@@ -684,3 +1085,203 @@ button:hover {
   color: #fff;
   background-color: rgba(96, 112, 255, 1);
 }
+
+@media screen and (min-width: 600px) {
+  .about-myself {
+    display: flex;
+    flex-direction: row;
+    height: 668px;
+    width: 100%;
+    margin-top: 0%;
+  }
+
+  .top-section {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
```

Comment:
- [ ] Your page is not entirely responsive. Your mobile version should still be as it was before. Here's your page at 650px.
![Screenshot_2021-09-23 Portfolio](https://user-images.githubusercontent.com/65501535/134482277-9f5ceaa0-e322-409a-aa3d-a16b85d79174.png)

-